docs/_publish/recommended-integration-features.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Recommended Integration Features
```

Comment:
I have no context of where this page comes from, but this is great guidance!

Reply:
It's a project I worked on on behalf of the Partnerships team during deep focus week! I have more project details linked in the PR description if you're interested.